stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/ChannelListView.kt

```diff
@@ -368,10 +368,22 @@ public class ChannelListView : FrameLayout {
         this.loadingView.isVisible = true
     }
 
+    @Deprecated(
+        message = "Deprecated in favor of new logic for setChannels. To set the loading item call set channels with" +
+            "the loading item already in the list. Show loading more function is race condition prone, which can" +
+            "cause the list to apply an old state of channels.",
+        level = DeprecationLevel.WARNING
+    )
     public fun showLoadingMore() {
         this.simpleChannelListView.showLoadingMore(true)
     }
 
+    @Deprecated(
+        message = "Deprecated in favor of new logic for setChannels. To set the loading item call set channels with" +
+            "the loading item already in the list. Show loading more function is race condition prone, which can" +
+            "cause the list to apply an old state of channels.",
```

Comment:
Message here doesn't make sense - we should set the list _without_ the item, right?

Reply:
Also references `showLoadingMore`